examples/variational_inference/empirical-approx-overview.ipynb

```diff
@@ -18,11 +18,13 @@
      "name": "stdout",
```

Comment:
<p>This looks odd. I think more recent arviz would plot them better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This is probably right, we improved the kde a couple releases back, the original implementation was too smoothing and did quite a bad job for multimodal and mixture distributions. The PR with the changes has some comparisons: https://github.com/arviz-devs/arviz/pull/1284</p>

Reply:
<p>Sounds great, thanks for the reference. Updated arviz dependency now combines the two into 1 graph.</p>